common/button/button-en.html

```diff
@@ -0,0 +1,426 @@
+---
+{
+	"title": "Buttons",
+	"language": "en",
+	"altLangPage": "button-fr.html",
+	"description": "Buttons page including working examples to test how various button styles appear and function",
+	"dateModified": "2023-06-14"
+}
+---
+
+<h2>On this page:</h2>
+
+<ul>
+	<li><a href="#purpose">Purpose</a></li>
+	<li><a href="#basic">Basic use</a></li>
+	<li><a href="#enhanced">Enhanced use</a></li>
+	<li><a href="#colours/styles">Colours/Styles</a></li>
+	<li><a href="#calltoaction">Call to Action</a></li>
+	<li><a href="#sizing">Sizing</a></li>
+	<li><a href="#fullwidth">Full Width</a></li>
+	<li><a href="#activestate">Active State</a></li>
+	<li><a href="#disabledstate">Disabled State</a></li>
+	<li><a href="#bghorizontal">Button Groups (Horizontal)</a></li>
+	<li><a href="#bgvertical">Button Groups (Vertical)</a></li>
+	<li><a href="#buttontoolbar">Button Toolbar</a></li>
+	<li><a href="#inputgroups">Input Groups</a></li>
+	<li><a href="#datatoggle">Data-toggle Buttons</a></li>
+</ul>
+
+<h2>Button vs links</h2>
+<dl class="dl-horizontal">
+<dt>Button</dt>
+<dd>A button will perform an action on the current page without moving the focus or without navigating the user away</dd>
+<dt>Link</dt>
+<dd>A link will send the user at a location, in the same page via an anchor (`#`) link or to an another page without sending any data over.</dd>
+</dl>
+<p>Do not use the role button on an anchor, use a button element instead.</p>
+
+<h2 id="basic">Basic use (Default)</h2>
+
+<h3>Working example</h3>
+<h4>Scaffolding (Default):</h4>
+<button type="submit">Button</button>
+<input type="button" value="Input" />
+<input type="submit" value="Submit" />
+<a href="#" >Link</a>
+
+<h4>With the default style:</h4>
+<button class="btn btn-default" type="submit">Button</button>
+<input class="btn btn-default" type="button" value="Input" />
+<input class="btn btn-default" type="submit" value="Submit" />
+<a class="btn btn-default" href="#" role="button">Link</a>
+
+<h4>Code Sample</h4>
+<pre><code>
+	// Default button:
+	&lt;button type="submit"&gt;
+	Button
+	&lt;/button&gt;
+	&lt;input type="button" value="Input"/&gt;
+	&lt;input type="submit" value="Submit"/&gt;
+	&lt;a href="#" role="button"&gt;
+	Link
+	&lt;/a&gt;
+
+	// Styled button:
+	&lt;button class="<strong>btn btn-default</strong>" type="submit"&gt;
+	Button
+	&lt;/button&gt;
+	&lt;input class="<strong>btn btn-default</strong>" type="button" value="Input"/&gt;
+	&lt;input class="<strong>btn btn-default</strong>" type="submit" value="Submit"/&gt;
+	&lt;a class="<strong>btn btn-default</strong>" href="#" role="button"&gt;
+	Link
+	&lt;/a&gt;
+</code></pre>
+
+<h2 id="enhanced">Enhanced use</h2>
+
+<h3 id="colours/styles">Colours/Styles</h3>
+<p>Colours <strong>signify</strong> various things such as a link to warning information (caution styled button). The text of that styled button will also contains word describing the state like "Warning notice".</p>
+<h4>Working example</h4>
```

Comment:
J'enlèverais le grouping "Enhanced use".

```suggestion
<h2 id="colours/styles">Colours/Styles</h2>
<p>Colours <strong>signify</strong> various things such as a link to warning information (caution styled button). The text of that styled button will also contains word describing the state like "Warning notice".</p>
<h3>Working example</h3>
```